tests/lib/Service/Event/SuggestionServiceTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Search\Service\Event;
+
+use Ibexa\Contracts\Search\Event\AfterSuggestionEvent;
+use Ibexa\Contracts\Search\Event\BeforeSuggestionEvent;
+use Ibexa\Contracts\Search\Model\Suggestion\SuggestionCollection;
+use Ibexa\Contracts\Search\Service\SuggestionServiceInterface;
+use Ibexa\Search\Model\SuggestionQuery;
+use Ibexa\Search\Service\Event\SuggestionService;
+use PHPUnit\Framework\TestCase;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+final class SuggestionServiceTest extends TestCase
+{
+    public function testSuggestion(): void
+    {
+        $eventDispatcherMock = $this->getEventDispatcherMock();
+        $eventDispatcherMock
+            ->method('dispatch')
+            ->withConsecutive(
```

Comment:
this 'withConsecutive' method is deprecated

Reply:
Not in the current version of PHPUnit.

Reply:
We should avoid to use any 3rd party code that we know is deprecated whenever possible. An alternative is to either increase the technological debt or revisit the whole codebase on each release of each vendor package. To me, neither of those sounds like a way to go. 😉

Reply:
It would be prudent to keep PHPUnit updated and promptly address any deprecations to maintain code quality and consistency.